flow/layers/layer.h

```diff
@@ -163,15 +163,39 @@ class Layer {
     needs_system_composite_ = value;
   }
 
+  // Returns the unclipped paint bounds in the layer's local coordinate
```

Comment:
`unclipped` seems inaccurate. The paint bounds should consider all clips in the layer subtree, but ignore all clips or cull rects of its ancestors?

Reply:
True.

Reply:
new comment pushed.